test/unit/TestSessionServiceTest.php

```diff
@@ -47,19 +48,25 @@
 class TestSessionServiceTest extends TestCase
 {
 
-    public function testGetTestSession()
+    public function testGetTestSessionLoadsNewSession()
```

Comment:
I guest that this test is `testGetTestSessionLoadsSameSession`

Reply:
Fixed

Reply:
Fixed